src/GitTabExpansion.ps1

```diff
@@ -537,10 +538,21 @@ function WriteTabExpLog([string] $Message) {
 
 if (!$UseLegacyTabExpansion -and ($PSVersionTable.PSVersion.Major -ge 6)) {
     $cmdNames = "git","tgit","gitk"
+
+    # Create regex pattern from $cmdNames: ^(git|git\.exe|tgit|tgit\.exe|gitk|gitk\.exe)$
+    $cmdNamesPattern = "^($(($cmdNames | ForEach-Object { "${_}|${_}\.exe" }) -join '|'))$"
```

Comment:
Simpler to just optionally match `.exe` at the end?
```suggestion
    $cmdNamesPattern = "^($($cmdNames -join '|'))(\.exe)?$"
```

Reply:
Done